docs/vsphere-golden-image.md

```diff
@@ -0,0 +1,172 @@
+
+In addition, you can run the provided PowerShell script [install-docker.ps1](vsphere_ci/scripts/install-docker.ps1) to programatically install `docker` in the Windows VM.
+
+
+### Pull required Windows container base images
+
+Pull all of the required Windows container base images needed for your applications. Such images are kernel-dependent
+on the Windows Server version you are using. See [Microsoft’s documentation](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-base-images) on pulling Windows container base images 
+for more information and to determine which images you need.
+
+To display the current Windows OS version using PowerShell
+
+```powershell
+    Get-ComputerInfo | Select OsHardwareAbstractionLayer 
+```
+
+Example base images:
+- Windows Server 1909 SAC: `docker pull mcr.microsoft.com/windows/servercore:1909`
+- Windows Server 2004 SAC: `docker pull mcr.microsoft.com/windows/servercore:2004`
+
+For additional Windows Server Core available images see the [Docker Hub repository](https://hub.docker.com/_/microsoft-windows-servercore?tab=description).
+
+
+## 5. Setup incoming connection for container logs
+
+Create a new firewall rule in the Windows VM to allow incoming connections for Container Logs, usually 
+on TCP port `10250` by running the following PowerShell command:
+
+```powershell
+    New-NetFirewallRule -DisplayName "ContainerLogsPort" -LocalPort 10250 -Enabled True -Direction Inbound -Protocol TCP -Action Allow -EdgeTraversalPolicy Allow
+```
+
+In addition, you can run the provided PowerShell script [install-firewall-rules.ps1](vsphere_ci/scripts/install-firewall-rules.ps1) to programatically create the container logs rule along with the OpenSSH incoming connections rule.
+
+
+## 6. Generalize the virtual machine installation
+
+To deploy the Windows VM as a reusable image, you have to first generalize the VM removing computer-specific information 
+such as installed drivers. The `Sysprep` with an *unattend* answer file generalize the image and make it ready for 
+deployment, maintaining all the changes needed for the WMCO installation. 
+
+You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example 
+and it **should not** be used directly. You must customize the example to fit your needs.
+
+To execute the `Sysprep` command use:
+
+```cmd
+    C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown /unattend:<path/to/unattend.xml>
+```
+
+where `<path/to/unattend.xml>` is the path to the customized answer file.
+
+> There is [a limit](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation#limits-on-how-many-times-you-can-run-sysprep) on how many times you can run Sysprep 
```

Comment:
Please add or `Note:` or something to that affect rather than `>` which implies it was quoted.